image.sparklyr.yaml

```diff
@@ -0,0 +1,39 @@
+name: "radanalyticsio/radanalytics-sparklyr"
+version: 1.0
+release: dev
+from: "rimolive/r-34-centos7:latest"
+user: 185
+labels:
+  - name: "io.k8s.description"
+    value: "Platform for building a radanalytics sparklyr app"
+  - name: "io.k8s.display-name"
+    value: "radanalytics sparklyr"
+  - name: "io.openshift.expose-services"
+    value: "8080:http"
+  - name: "io.openshift.s2i.scripts-url"
+    value: "image:///usr/libexec/s2i"
+  - name: "io.openshift.tags"
+    value: "builder,radanalytics,sparklyr"
+  - name: "io.radanalytics.sparkdistro"
+    value: "https://archive.apache.org/dist/spark/spark-2.2.0/spark-2.2.0-bin-hadoop2.7.tgz"
```

Comment:
the other s2i images are currently at 2.2.1, and will move to 2.3.0 in the next release